test/unit/config/config-location.test.js

```diff
@@ -61,48 +60,41 @@ tap.test('when overriding the config file location via NEW_RELIC_HOME', (t) => {
     await fsPromises.rm(NOPLACEDIR, { recursive: true })
   })
 
-  t.test('should load the configuration', (t) => {
-    t.doesNotThrow(() => {
+  await t.test('should load the configuration', (t, end) => {
+    assert.doesNotThrow(() => {
       Config.initialize()
+      end()
     })
-
-    t.end()
   })
 
-  t.test('should export the home directory on the resulting object', (t) => {
+  await t.test('should export the home directory on the resulting object', () => {
     const configuration = Config.initialize()
-    t.equal(configuration.newrelic_home, DESTDIR)
-
-    t.end()
+    assert.equal(configuration.newrelic_home, DESTDIR)
   })
 
-  t.test('should ignore the configuration file completely when so directed', (t) => {
-    process.env.NEW_RELIC_NO_CONFIG_FILE = 'true'
-    process.env.NEW_RELIC_HOME = '/xxxnoexist/nofile'
-
-    t.teardown(() => {
-      delete process.env.NEW_RELIC_NO_CONFIG_FILE
-      delete process.env.NEW_RELIC_HOME
-    })
-
-    let configuration
-
-    t.doesNotThrow(() => {
-      configuration = Config.initialize()
-    })
+  await t.test('should ignore the configuration file completely when so directed', (t, end) => {
+    try {
```

Comment:
I believe you've opted for a `try/finally` because `t.teardown` doesn't exist. But it actually does:

```js
t.after(() => {
	delete process.env.thing
})
``